_episodes/markdown.md

```diff
@@ -142,6 +142,30 @@ Let's do an exercise to try out writing more markdown.
 > {: .solution }
 {: .challenge }
 
+> ## Reference-style links
+> 
+> Up to now, we have used `inline style links` for example:
+>
+> ~~~
+> [Carpentries Webpage](https://carpentries.org/)
+> ~~~
+> {: .language-markdown }
+>
+> If you use a link more than once, consider using so called `reference-style links` instead.
+> Unlike inline-style links which have the URL inline with the description text, reference-style links
+> reference the URL via a reference.
```

Comment:
We might be using "reference" too much in this sentence?
```suggestion
> reference the URL via a label.
```
"Label" is used [here](https://www.markdownguide.org/basic-syntax/#reference-style-links) which seems like a good decription.  Maybe we should link this page somewhere in this call out?